language/stdlib/modules/libra_account.move

```diff
@@ -156,12 +156,19 @@ module LibraAccount {
         ) {
             // sanity check of signature validity
             Transaction::assert(Vector::length(&metadata_signature) == 64, 9001);
+            // message should be metadata | sender_address | amount | domain_separator
+            // separator is the UTF8-encoded string @$LIBRA_ATTEST$@
+            let domain_separator = x"2240244c494252415f4154544553544022";
```

Comment:
I believe `x"2240244c494252415f4154544553544022"` is the utf-8 form of 
```
"@$LIBRA_ATTEST@"
```

`40244c494252415f4154544553542440` should be the one for `@$LIBRA_ATTEST@`

Reply:
Good catch, will fix!

I also just realized that LCS strings have a length prefix, which means we might want to use one here... thoughts?

Reply:
Btw in Libra we use "@@$$LIBRA$$@@", just for consistency maybe we should change that or this with single or double symbols?

Reply:
Accidentally let this land, but will fix in a follow-up + add the length prefix (which it sounds like we need)

> Btw in Libra we use "@@$$LIBRA$$@@", just for consistency maybe we should change that or this with single or double symbols?

I think whether consistency is a virtue or not here is an interesting question... I was actually trying to use a `@$` convention different from Libra to avoid conflicts with any future Libra domain specifiers. But happy to change this to whatever